### PR TITLE
fix: license disable as bool

### PR DESCRIPTION
**Pull request by jokimina**
image

image

https://docs.lemonsqueezy.com/api/license-keys#the-license-key-object

The examples and test data in the official documentation are all of type bool.

### CODE REVIEW

**Comment by AchoArnold** [comment]
@jokimina 

Please fix the failing unit tests. You can run them with the `go test` command.

Thanks.

File: license_key.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ type LicenseKeyAttributes struct {
 	KeyShort        string      `json:"key_short"`
 	ActivationLimit int         `json:"activation_limit"`
 	InstancesCount  int         `json:"instances_count"`
-	Disabled        int         `json:"disabled"`
+	Disabled        bool        `json:"disabled"`
```
Comment (AchoArnold) [inline license_key.go:18]: Fix the `license_key_service_test`

Comment (jokimina) [reply license_key.go:18]: Thank you for the reminder, the tests have been fixed. I'll be more careful next time 😂

**Comment by AchoArnold** [comment]
Thanks

**Pull request merged**